draft-ietf-tls-subcerts.md

```diff
@@ -458,8 +462,12 @@ extension is not present, the client MUST NOT send a delegated credential.
 When a (D)TLS version negotiated is less than 1.3, the client MUST ignore this extension.
 
 The client MUST send the delegated credential as an extension in the
-CertificateEntry of its end-entity certificate; the server SHOULD ignore
-delegated credentials sent as extensions to any other certificate.
+CertificateEntry of its end-entity certificate; the server MUST NOT use
+delegated credentials sent as extensions to any other certificate, and
+SHOULD ignore them, but MAY abort the handshake with an "illegal_parameter"
+alert. If the client sends multiple delegated credentials extensions
+in a single CertificateEntry, the server MUST abort the handshake with
+an "illegal_parameter" alert.
```

Comment:
while this is technically a change that the WG should approve, as it is fairly obvious the additional restrictions were implied before, I approve of this change.